src/main.cpp

```diff
@@ -28,5 +29,7 @@ int main(int argc, char **argv)
     setenv("XDG_CACHE_HOME", appCachePath.toUtf8().constData(), 1);
 
     LunaQmlApplication application(argc, argv);
+
+    QtWebEngine::initialize();
```

Comment:
That looks a bit suspicious. What would be with applications using qmlscene and pure QML? Is that intended by upstream?